docs/antrea-gke.md

```diff
@@ -0,0 +1,128 @@
+# Deploying Antrea on a GKE cluster
+
+We support running Antrea inside of GKE clusters on Ubuntu host. Antrea would operate
+in noEncap mode, in which no encapsulation is required for any kind of traffic (Intra node,
+Inter node etc) and enforcing network policies using OVS.
+
+For details on Antrea noEncap Mode, refer to this. [TODO]
+
+## GKE Prerequisites
+
+1. Install the Google Cloud SDK (gcloud). Refer to this (https://cloud.google.com/sdk/install)
+
+    ```bash
+    curl https://sdk.cloud.google.com | bash
+    ```
+
+2. Make sure you are authenticated to use the Google Cloud API
+
+    ```bash
+    export ADMIN_USER=user@email.com
+    gcloud auth login
+    ```
+
+3. Create a project or use an existing one
+
+    ```bash
+    export GKE_PROJECT=gke-clusters
+    gcloud projects create $GKE_PROJECT
+    ```
+
+## Creating the cluster
+
+You can use any method to create a GKE cluster (gcloud SDK, gcloud Console etc). The example
+given here is using the Google Cloud SDK.
+
+Note: Antrea is supported on Ubuntu hosts only for GKE cluster. Also, its must to select service
+CIDR at the time of cluster deployment.
+
+1. Create a GKE cluster
+
+    ```bash
+    export GKE_ZONE="us-west1"
+    export GKE_HOST="UBUNTU"
+    export GKE_SERVICE_CIDR="10.94.0.0/16"
+    gcloud container --project $GKE_PROJECT clusters create cluster1 --image-type $GKE_HOST \
+       --zone $GKE_ZONE --enable-ip-alias --services-ipv4-cidr $GKE_SERVICE_CIDR
+    ```
+
+2. Access your cluster
+
+    ```bash
+    kubectl get nodes
+    NAME                                      STATUS   ROLES    AGE     VERSION
+    gke-cluster1-default-pool-93d7da1c-61z4   Ready    <none>   3m11s   v1.14.10-gke.17
+    gke-cluster1-default-pool-93d7da1c-rkbm   Ready    <none>   3m9s    v1.14.10-gke.17
+    ```
+
+3. Create a cluster-admin-binding
+
+    ```bash
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user user@email.com
+    ```
+
+    Note: To create clusterRoleBinding, the user must have `container.clusterRoleBindings.create` permission.
+Use this command to enable it, if the previous command fails due to permission error. Only cluster Admin can
+assign this permission.
+
+    ```bash
+    gcloud projects add-iam-policy-binding $GKE_PROJECT --member user:user@email.com --role roles/container.admin
+    ```
+
+## Deploying Antrea
+
+1. Prepare the Cluster Nodes
+
+    Deploy antrea-node-init DaemonSet to enable kubelet to operate in CNI mode.
+
+    ```bash
+    kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.5.0/antrea-node-init.yml
```

Comment:
better to use <TAG> here instead of 0.5.0

Reply:
Will keep it to 0.5.0. This wont change per release.